Client/Frontend/Menu/HomeMenuController.swift

```diff
@@ -92,7 +92,12 @@ class HomeMenuController: UIViewController, PopoverContentComponent {
     
     bookmarksController.linkNavigationDelegate = self
     bookmarksController.bookmarksDidChange = { [weak self] in
-      self?.updateBookmarkStatus()
+        self?.updateBookmarkStatus()
+        if self?.bookmarksController.bookmarksFRC?.fetchedObjects?.count ?? 0 < 1 {
```

Comment:
See above comment about this too.

Reply:
I am worried if I messed up the tabulation size here. Is it ok for me to continue using tab size of 4 in this file, because it was 4 in other files?

Reply:
If I understood correctly, the `?` is not necessary here. But when I tried removing it I got following error message. So should I leave it?

> Value of optional type 'HomeMenuController?' must be unwrapped to refer to member 'bookmarksController' of wrapped base type 'HomeMenuController'
Chain the optional using '?' to access member 'bookmarksController' only for non-'nil' base values
Force-unwrap using '!' to abort execution if the optional value contains 'nil'